### PR TITLE
Register-ScheduledTask: Three fixes

### PR DESCRIPTION
**Pull request by skycommand**
1. Re-wrote a sentence to eliminate the following **arcane** phrases: "Win16 applications", "OS/2 applications", and "MS-DOS applications". These phrases were used [in a sense other than their direct, literal meaning](https://en.wikipedia.org/wiki/Metonymy). Understanding these phrases was in no way essential to understanding this document.
2. Fixed a rather amusing bug. The following part of the sentence was rendered in italics:

    > `*.bat), command files (*`

    The reason is the asterisk characters. These characters are part of the Markdown syntax. Of course, using them in this context is semantically wrong too; this cmdlet does not support wildcards.
3. Synced all three versions of the Register-ScheduledTask documentation pages. Before doing so, I verified that all three versions of Windows accept the same parameter sets.

### CODE REVIEW

**Comment by e0i** [comment]
Closing and re-opening to refresh validation checks.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012r2-ps/scheduledtasks/Register-ScheduledTask.md [hunk 1]
```diff
@@ -116,7 +119,7 @@ Accept wildcard characters: False
 
 ### -CimSession
 Runs the cmdlet in a remote session or on a remote computer.
-Enter a computer name or a session object, such as the output of a New-CimSessionhttp://go.microsoft.com/fwlink/p/?LinkId=227967 or Get-CimSessionhttp://go.microsoft.com/fwlink/p/?LinkId=227966 cmdlet.
+Enter a computer name or a session object, such as the output of a [New-CimSession](https://docs.microsoft.com/en-us/powershell/module/cimcmdlets/new-cimsession) or [Get-CimSession](https://docs.microsoft.com/en-us/powershell/module/cimcmdlets/get-cimsession) cmdlet.
```
Comment (get-itips) [inline docset/winserver2012r2-ps/scheduledtasks/Register-ScheduledTask.md:122]:
```suggestion
Enter a computer name or a session object, such as the output of a [New-CimSession](https://docs.microsoft.com/powershell/module/cimcmdlets/new-cimsession) or [Get-CimSession](https://docs.microsoft.com/powershell/module/cimcmdlets/get-cimsession) cmdlet.
```

File: docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md [hunk 2]
```diff
@@ -106,7 +111,7 @@ Accept wildcard characters: False
 
 ### -CimSession
 Runs the cmdlet in a remote session or on a remote computer.
-Enter a computer name or a session object, such as the output of a New-CimSessionhttp://go.microsoft.com/fwlink/p/?LinkId=227967 or Get-CimSessionhttp://go.microsoft.com/fwlink/p/?LinkId=227966 cmdlet.
+Enter a computer name or a session object, such as the output of a [New-CimSession](https://docs.microsoft.com/en-us/powershell/module/cimcmdlets/new-cimsession) or [Get-CimSession](https://docs.microsoft.com/en-us/powershell/module/cimcmdlets/get-cimsession) cmdlet.
```
Comment (get-itips) [inline docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md:114]:
```suggestion
Enter a computer name or a session object, such as the output of a [New-CimSession](https://docs.microsoft.com/powershell/module/cimcmdlets/new-cimsession) or [Get-CimSession](https://docs.microsoft.com/powershell/module/cimcmdlets/get-cimsession) cmdlet.
```

File: docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md [hunk 3]
```diff
@@ -16,57 +16,62 @@ Registers a scheduled task definition on a local computer.
...
+ [[-TaskName] <String>] [[-TaskPath] <String>] [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
+ [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 The **Register-ScheduledTask** cmdlet registers a scheduled task definition on a local computer.
 
-You can register a task to run any of the following application or file types: Win32 applications, Win16 applications, OS/2 applications, MS-DOS applications, batch files (*.bat), command files (*.cmd), or any properly registered file type.
+You can register a task to run executable files (`.exe` and `.com`), batch files (`.bat` and `.cmd`),
+or any registered file type. However, this cmdlet does not check whether the file you intend it to
```
Comment (get-itips) [inline docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md:52]: can you please review the new lines here? Thank you

Comment (skycommand) [reply docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md:52]: Oh, this change is the main point of this whole PR. To quote myself:

> Re-wrote a sentence to eliminate the following **arcane** phrases: "Win16 applications", "OS/2 applications", and "MS-DOS applications". These phrases were used [in a sense other than their direct, literal meaning](https://en.wikipedia.org/wiki/Metonymy). Understanding these phrases was in no way essential to understanding this document.

As you can probably understand from the context, these phrases are supposed to refer to "properly registered file type". The problem is that they don't. The reader needs to know a lot of history and arcana before discovering that they are wrong.

For example, I suspect "MS-DOS applications" is a metonym for `.com` files. Of course, MS-DOS apps could have `.exe` extensions too. Had it been Windows 9x or Windows 3.1, the phrase could refer to `.pif` files as well. Sure, you can go learn about NTVDM, NTIO.SYS and NTDOS.SYS, but once did, you discover that all this problematic paragraph intended to say was:

> You can register a task to run any properly registered file type.

**Review by dariomws — APPROVED** [review]

File: docset/windows/scheduledtasks/Register-ScheduledTask.md [hunk 4]
```diff
@@ -57,7 +57,9 @@ Register-ScheduledTask [-Force] [-InputObject] <CimInstance> [[-Password] <Strin
 ## DESCRIPTION
 The **Register-ScheduledTask** cmdlet registers a scheduled task definition on a local computer.
 
-You can register a task to run any of the following application or file types: Win32 applications, Win16 applications, OS/2 applications, MS-DOS applications, batch files (*.bat), command files (*.cmd), or any properly registered file type.
+You can register a task to run executable files (`.exe` and `.com`), batch files (`.bat` and `.cmd`),
+or any registered file type. However, this cmdlet does not check whether the file you intend it to
+run is compatible with your version, edition or platform specialization of Windows.
```
Comment (JohanFreelancer9) [inline docset/windows/scheduledtasks/Register-ScheduledTask.md:62]:
```suggestion
run is compatible with your version, edition, or platform specialization of Windows.
```

File: docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md [hunk 5]
```diff
@@ -16,57 +16,62 @@ Registers a scheduled task definition on a local computer.
...
+run is compatible with your version, edition or platform specialization of Windows.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md:53]:
```suggestion
run is compatible with your version, edition, or platform specialization of Windows.
```

File: docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md [hunk 6]
```diff
@@ -327,16 +332,19 @@ Specifies the XML string that contains a task definition.
...
 Required: True
-Position: 3
+Position: 2
 Default value: None
 Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md:346]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2012r2-ps/scheduledtasks/Register-ScheduledTask.md [hunk 7]
```diff
@@ -56,7 +56,9 @@ Register-ScheduledTask [-Force] [-InputObject] <CimInstance> [[-Password] <Strin
 ## DESCRIPTION
 The **Register-ScheduledTask** cmdlet registers a scheduled task definition on a local computer.
 
-You can register a task to run any of the following application or file types: Win32 applications, Win16 applications, OS/2 applications, MS-DOS applications, batch files (*.bat), command files (*.cmd), or any properly registered file type.
+You can register a task to run executable files (`.exe` and `.com`), batch files (`.bat` and `.cmd`),
+or any registered file type. However, this cmdlet does not check whether the file you intend it to
+run is compatible with your version, edition or platform specialization of Windows.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/scheduledtasks/Register-ScheduledTask.md:61]:
```suggestion
run is compatible with your version, edition, or platform specialization of Windows.
```

File: docset/winserver2012r2-ps/scheduledtasks/Register-ScheduledTask.md [hunk 8]
```diff
@@ -349,7 +351,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/scheduledtasks/Register-ScheduledTask.md:354]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - just a few minor edits, please.

**Comment by skycommand** [comment]
@JohanFreelancer9 Actually, let me do it the right way, and for all three pages too.

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @skycommand.

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you @skycommand.

File: docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md (hunk 3 above)
Comment (get-itips) [reply docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md:52]: I was referring to the carrier return that looked strange. Seems that it is fixed now.

**Review by get-itips — APPROVED** [review]

File: docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md (hunk 3 above)
Comment (skycommand) [reply docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md:52]: Huh! Well, if it is fixed now... I'll mark this as resolved.

**Pull request merged**